packages/shared/styles/components/atoms/SfProperty.scss

```diff
@@ -2,45 +2,38 @@
 
 .sf-property {
   display: flex;
-  align-items: center;
   color: var(--property-color, var(--c-text));
   &__name {
-    display: flex;
-    align-items: center;
-    margin: var(--property-name-margin, 0 var(--spacer) 0 0);
-    color: var(--property-name-color, var(--c-text-muted));
-    font: var(
-      --property-name-font,
-      font(
-        --property-name,
-        400,
-        var(--font-size-regular),
-        1.6,
-        var(--body-font-family-secondary)
-      )
-    );
+    margin: var(--property-name-margin, 0 var(--spacer-extra-small) 0 0);
+    color: var(--property-name-color, var(--c-secondary-variant));
     text-transform: var(--property-name-text-transform);
     &::after {
       content: var(--property-name-content, ":");
     }
   }
-  &__value {
-    font: var(
-      --property-value-font,
-      font(
-        --property-value-font,
-        400,
-        var(--font-size-regular),
-        1.6,
-        var(--body-font-family-secondary)
-      )
-    );
-  }
+  @include font(
+    --property-font,
+    var(--font-weight-regular),
+    var(--font-size-extra-small),
+    1.2,
```

Comment:
just a question - why 1.2? :)

Reply:
because I took the propotion between line-height;26px(mobile:14px) and font-size: 22px(mobile:12px) which is closer to 1.2. Maybe I also ask Marcin about that?

Reply:
Marcin says it should be 1.2 in this case.